modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -212,6 +212,17 @@ public void execute() throws MojoExecutionException {
             }
         }
 
+        File ignoreClassesFile =
+            new File(outputDirectory, "ModernizerIgnoreAnnotatedClasses.txt");
+        if (ignoreClassesFile.exists()) {
+            Collection<String> ignoreClasses =
+                readFile(ignoreClassesFile.toString());
```

Comment:
Are we really hitting the 80 char limit here?

Reply:
Sadly, yes